### PR TITLE
Added version filtering in find dependencies

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/find_dependencies/find_dependencies.py [hunk 1]
```diff
@@ -164,17 +168,16 @@ def _search_packs_by_items_names(items_names, items_list, exclude_ignored_depend
...
-        if content_items:
-            pack_names = list(map(lambda s: next(iter(s.values()))['pack'], content_items))
-            if not exclude_ignored_dependencies:
-                return set(pack_names)
-            return {p for p in pack_names if p not in constants.IGNORED_DEPENDENCY_CALCULATION}
+        pack_names = set()
+        for item in items_list:
+            item_details = list(item.values())[0]
+            if item_details.get('name', '') in items_names and 'pack' in item_details and \
+                    LooseVersion(item_details.get('toversion', '99.99.99')) >= MINIMUM_DEPENDENCY_VERSION:
```
Comment (Itay4) [inline demisto_sdk/commands/find_dependencies/find_dependencies.py:175]: what about items that have `toVersion` ?

Comment (yaakovi) [reply demisto_sdk/commands/find_dependencies/find_dependencies.py:175]: this is from the id set, everything is `toversion` (no capital V)

File: demisto_sdk/commands/find_dependencies/find_dependencies.py [hunk 2]
```diff
@@ -164,17 +168,16 @@ def _search_packs_by_items_names(items_names, items_list, exclude_ignored_depend
...
+                pack_names.add(item_details.get('pack'))
 
-        return None
```
Comment (IkaDemisto) [inline demisto_sdk/commands/find_dependencies/find_dependencies.py:177]: Just update the return value in doc string in such case, may be confusing later.

File: demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py [hunk 3]
```diff
@@ -178,6 +178,41 @@ def test_collect_scripts_depends_on_two_scripts(self, id_set):
...
+        """
+        Given
+            - A script entry in the id_set depending on QRadar command.
+
+        When
+            - Building dependency graph for pack.
+
+        Then
+            - Extracting the packs that the script depends on.
+            - Should ignore the Deprecated pack due to toversion settings of old QRadar integration.
```
Comment (IkaDemisto) [inline demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py:191]: Should ignore Deprecated packs or `NonSupported` pack?

Comment (yaakovi) [reply demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py:191]: deprecated, old integration is there

**Pull request merged**